lib/ClangImporter/ImportType.cpp

```diff
@@ -3247,9 +3250,13 @@ ImportedType ClangImporter::Implementation::importAccessorParamsAndReturnType(
   DeclContext *origDC = importDeclContextOf(property,
                                             property->getDeclContext());
   assert(origDC);
+  auto fieldType = isGetter ? clangDecl->getReturnType() : clangDecl->getParamDecl(0)->getType();
```

Comment:
Can you add a test case for when this is not a getter?

Reply:
Do you mean when it is *only* a getter? I thought `// CHECK-NEXT:   class func setBar(_ bar: Bar)` was testing for the setter.